src/commands/sourcemaps/upload.rs

```diff
@@ -47,6 +47,13 @@ pub fn make_command(command: Command) -> Command {
                 .value_parser(validate_distribution)
                 .help("Optional distribution identifier for the sourcemaps."),
         )
+        .arg(
+            Arg::new("note")
+                .long("note")
+                .short('n')
```

Comment:
If this will be used very sparingly, I'd remove `n` shortcut for this end be explicit about the flag. It's only 4 characters anyway.